inc/blocks/ab-test/edit.css

```
@@ -0,0 +1,13 @@
+button.ab-test-xb-preview__tab {
+	background-color: #4667de;
```

Basic theme styles might set other background properties like background-image so best to just completely clobber them using `background`:
```suggestion
	background: #4667de;
```